File: Videos1_20/PythonCode/FormEvent.py
class FormEvent:
    def __init__(self, source, name, occupation, ageCat, empCat, taxId, usCitizen):
        self.source = source
        self.name = name
        self.occupation = occupation
        self.ageCategory = ageCat
        self.empCat = empCat
        self.taxId = taxId
        self.usCitizen = usCitizen

    def getName(self): return self.name
    def getOccupation(self): return self.occupation
    def getAgeCategory(self): return self.ageCategory
    def getEmploymentCategory(self): return self.empCat
    def getTaxId(self): return self.taxId
    def isUsCitizen(self): return self.usCitizen

File: Videos1_20/CSharpcode/FormEvent.cs
using System;

namespace CSharpCode {
    public class FormEvent : EventArgs {
        private string name;
        private string occupation;
        private int ageCategory;
        private string empCat;
        private string taxId;
        private bool usCitizen;

        public FormEvent(object source, string name, string occupation, int ageCat, string empCat, string taxId, bool usCitizen) {
            this.name = name;
            this.occupation = occupation;
            this.ageCategory = ageCat;
            this.empCat = empCat;
            this.taxId = taxId;
            this.usCitizen = usCitizen;
        }

        public string getName() { return name; }
        public string getOccupation() { return occupation; }
        public int getAgeCategory() { return ageCategory; }
        public string getEmploymentCategory() { return empCat; }
        public string getTaxId() { return taxId; }
        public bool isUsCitizen() { return usCitizen; }
    }
}